src/macros.cr

```diff
@@ -83,6 +83,25 @@ macro record(name, *properties)
   end
 end
 
+private record TypeSentinel
```

Comment:
[You are right](https://glot.io/snippets/esiky8isk4)

Sucks that it doesn't work and doesn't error